hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -864,6 +864,14 @@
       of max amount of pipelines which are OPEN.
     </description>
   </property>
+  <property>
```

Comment:
We the config is defined as it is in ECWriteableContainerProvider, using the "Type Safe Config" definitions, at compile time, any config with those annotations are added into a ozone-default-generated.xml file, so we don't need to update the ozone-default.xml with this value.

The problem we have, is that half the project uses the old way (SCMConfigKeys and ozone-default.xml) and half uses the new way. The project direction is to adopt the new way, although I personally prefer the old way, as it is simpler and easier to understand :-)

As the config is defined in the "new way" here, I think we should remove the additions to ozone-default.xml and ScmConfigKeys.

Reply:
Thanks for explaining. How do we change the config in the "new way"?
More specifically, how to set this value in MiniOzoneCluster?

Reply:
Example with `SCMClientConfig` can be found just a few lines below the current change in `MiniOzoneClusterImpl`.

https://github.com/apache/ozone/blob/88b5e854be89be7ef3944527be4b87d456ee297b/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneClusterImpl.java#L692-L695

Reply:
Thanks @sodonnel and @adoroszlai for the info.